ui/page/privacy/manual_mixer_setup_page.go

```diff
@@ -122,65 +116,110 @@ func (pg *ManualMixerSetupPage) OnNavigatedTo() {
-									})
+									return layout.Flex{Alignment: layout.Start}.Layout(gtx,
+										layout.Rigid(func(gtx C) D {
+											return pg.backFromManual.Layout(gtx)
+										}),
+										layout.Rigid(func(gtx C) D {
+											label := pg.Theme.H6(values.String(values.StrSetUpStakeShuffleAutoOrManualG))
+											return layout.Inset{
+												Top:  values.MarginPadding10,
+												Left: values.MarginPadding5,
+											}.Layout(gtx, label.Layout)
+										}),
+										// TODO: The Label above will not go to the left unless I eat up all the space
+										// on the right here. This cannot be the correct way to implement this. Find a
+										// way to remove this spacer.
+										layout.Rigid(func(gtx C) D {
+											return layout.Spacer{Width: 100000000}.Layout(gtx)
+										}),
+									)
 								}),
 								layout.Rigid(func(gtx C) D {
-									return layout.Inset{Top: values.MarginPaddingMinus15}.Layout(gtx, func(gtx C) D {
-										return pg.mixerAccountSections(gtx, values.String(values.StrUnmixedAccount), func(gtx layout.Context) layout.Dimensions {
-											return pg.unmixedAccountSelector.Layout(pg.ParentWindow(), gtx)
-										})
-									})
+									return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
+										layout.Rigid(func(gtx C) D {
+											return pg.mixerAccountSections(gtx, values.String(values.StrMixedAccount), func(gtx layout.Context) layout.Dimensions {
+												return pg.mixedAccountSelector.Layout(pg.ParentWindow(), gtx)
+											})
+										}),
+										layout.Rigid(func(gtx C) D {
+											return layout.Inset{Top: values.MarginPaddingMinus15}.Layout(gtx, func(gtx C) D {
+												return pg.mixerAccountSections(gtx, values.String(values.StrUnmixedAccount), func(gtx layout.Context) layout.Dimensions {
+													return pg.unmixedAccountSelector.Layout(pg.ParentWindow(), gtx)
+												})
+											})
+										}),
+										layout.Rigid(func(gtx C) D {
+											return layout.Spacer{Height: 10}.Layout(gtx)
+										}),
+										layout.Rigid(func(gtx C) D {
+											card := pg.Theme.Card()
+											card.Color = pg.Theme.Color.Gray4
+											return card.Layout(gtx, func(gtx C) D {
+												gtx.Constraints.Min.X = gtx.Constraints.Max.X
+												gtx.Constraints.Min.Y = 110
+												return layout.Inset{Top: values.MarginPadding10, Left: values.MarginPadding16,
+													Right: values.MarginPadding16}.Layout(gtx, func(gtx C) D {
+													return layout.Inset{Top: values.MarginPaddingMinus15, Left: values.MarginPaddingMinus15,
+														Right: values.MarginPaddingMinus15, Bottom: values.MarginPaddingMinus30}.Layout(gtx, func(gtx C) D {
+														return layout.Flex{Alignment: layout.Start}.Layout(gtx,
+															layout.Flexed(1, func(gtx C) D {
+																gtx.Constraints.Max.X = 40
+																// TODO: Find a way to center this vertically.
```

Comment:
Can't seem how to get a widget to be equidistant from the top and bottom of whatever the final size of the parent becomes. Nested flexed layouts don't seem to work as I would expect, although it's more likely I am implementing them wrong. Here and in a few other places.

Reply:
Can check if `Spacing: layout.SpaceAround` in the flex will work.

Reply:
Thanks, that and it seems you need to set the minimum Y to something big enough to fit everything, allowing it to center.